### ISSUE TITLE
Calendar: manualInput property ignored

### ISSUE THREAD

**Issue by dmhendricks** [primefaces/primevue#3523]
### Describe the bug

After upgrading from 3.20.0 to 3.22.0, the `manualInput` property on the [Calendar](https://www.primefaces.org/primevue/calendar) component is now being ignored.

```html
<Calendar v-model="someDate" :manualInput="true" />
```

### Reproducer

&nbsp;

### PrimeVue version

3.22.0

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vue CLI App

### Browser(s)

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36

### Steps to reproduce the behavior

1. npm update to PrimeVue 3.22.0
2. Try to manually input date/time on a Calendar control where manualInput paramter = true

### Expected behavior

In previous versions, I was able to manually edit the date/time in the Calendar control when the manualInput property is true.

**Comment by tugcekucukoglu** [comment]
default property of the `manualInput` is true. Please check your code again.

**Comment by HannesJoerk0174** [comment]
@tugcekucukoglu I probably have the same issue as dmhendricks. I know that manualInput is the default value but it just doesn't work anymore. Neither when leaving the default or explicitly setting it to true can I actually type values in the calendar input. When clicking the field input is only possible via clicking in the calendar but not by typing.

It also occurs in your documentation: https://primefaces.org/primevue/calendar

**Comment by Quent1L** [comment]
I have the same problem with my calendar with the time available. 
I confirm that it also appears in your documentation: https://primefaces.org/primevue/calendar

**Comment by dmhendricks** [comment]
I just tried 3.23.0 and this is still an issue. I can't update because users complain about not being able to manually edit the field, unless I find a different calendar picker (which I'd really rather avoid).

**Comment by tsofist** [comment]
Any updates? 😞

**Comment by xdghcnt** [comment]
I can't use current version documentation and current version bug fixes (e.g. DataTable row-style prop) because of this bug

**Comment by titou10titou10** [comment]
Same for me with primevue v3.26.1 and vue 3.2.32

    <Calendar
      v-model="dateOperation"
      date-format="yy-mm-dd"
      :number-of-months="2"
      class="flex-grow-0"
      :input-style="{ width: '7rem' }"
      :manual-input="true"
    />

**Comment by skywaving** [comment]
still not working properly :(

**Comment by Marlight** [comment]
It's only in calendar mode, in time-only works.

**Comment by Nikolai-Nikolov** [comment]
A workaround is to set :show-on-focus="false" and :show-icon="true". If it's not messing with the design requirements.

**Comment by knadich** [comment]
> any updates on this? does this get fixed?

Hello,
did you find the solution please ? I have the same problem
Thanks

**Comment by carstenmetzler** [comment]
Same as https://github.com/primefaces/primevue/issues/3624. Any updates on this?

**Comment by cagataycivici** [comment]
Set to be fixed at 3.34.2.
